src/app/(mobile-ui)/home/page.tsx

```diff
@@ -1,26 +1,36 @@
 'use client'
 
 import { ArrowIcon } from '@/components/0_Bruddle'
-import React from 'react'
-import { motion, useAnimation } from 'framer-motion'
-import { useRef, useEffect } from 'react'
-import classNames from 'classnames'
+import HomeHeader from '@/components/Home/HomeHeader'
 import { HomeLink } from '@/components/Home/HomeLink'
-import { useWallet } from '@/context/walletContext'
-import { useAuth } from '@/context/authContext'
 import PointsBanner from '@/components/Home/PointsBanner'
-import { useRouter } from 'next/navigation'
-import HomeHeader from '@/components/Home/HomeHeader'
 import { WalletCard } from '@/components/Home/WalletCard'
+import { useAuth } from '@/context/authContext'
+import { useWallet } from '@/context/walletContext'
+import classNames from 'classnames'
+import { motion, useAnimation } from 'framer-motion'
+import { useRouter } from 'next/navigation'
+import { useEffect, useRef, useState } from 'react'
 
 const cardWidth = 300
 const cardMargin = 16
 
+const BALANCE_VISIBILITY_KEY = 'peanut-balance-visibility'
```

Comment:
nit, blocking: this seems like wrong place to define this - check out updatePeanutPreferences.

We should have all user preferences together, not separate in various files or various localstorage keys

Reply:
my bad, missed this one, gonna fix it 🫡